### PR TITLE
Fix web component theming

### PR DESCRIPTION
**Pull request by loiswells97**
- Fires custom event when the theme changes, so the web component's host page can update the theme straight away.
- Removes rogue semi colon from web component loading state

### CODE REVIEW

File: src/containers/WebComponentLoader.jsx [hunk 1]
```diff
@@ -165,7 +165,7 @@ const WebComponentLoader = (props) => {
     </>
   ) : (
     <>
-      <p>{t("webComponent.loading")}</p>;
+      <p>{t("webComponent.loading")}</p>
```
Comment (conorriches) [inline src/containers/WebComponentLoader.jsx:168]: Yessssss

**Review by conorriches — DISMISSED** [review]
Nice!

File: src/components/Menus/Sidebar/SettingsPanel/ThemeToggle/ThemeToggle.test.js [hunk 2]
```diff
@@ -38,6 +44,12 @@ describe("When default theme is light mode and cookie unset", () => {
     expect(cookies.cookies.theme).toBe("dark");
   });
 
+  test("Fires theme updated custom event when button clicked", async () => {
+    const button = screen.getByText("sidebar.settingsMenu.themeOptions.dark");
+    fireEvent.click(button);
+    expect(themeUpdatedHandler).toHaveBeenCalled();
```
Comment (sra405) [inline src/components/Menus/Sidebar/SettingsPanel/ThemeToggle/ThemeToggle.test.js:50]: 💭 we want to test that light/dark is distinguishable too right?

File: src/containers/WebComponentLoader.jsx (hunk 1 above)
Comment (sra405) [reply src/containers/WebComponentLoader.jsx:168]: 😮 that's where it is!!

File: src/components/Menus/Sidebar/SettingsPanel/ThemeToggle/ThemeToggle.test.js (hunk 2 above)
Comment (loiswells97) [reply src/components/Menus/Sidebar/SettingsPanel/ThemeToggle/ThemeToggle.test.js:50]: I actually tried to do this and it wasn't having it :/ Maybe worth a huddle this afternoon to see if we can get these tests to be more specific

Comment (loiswells97) [reply src/components/Menus/Sidebar/SettingsPanel/ThemeToggle/ThemeToggle.test.js]: Right, have refactored slightly and got this working 🎉

**Review by sra405 — APPROVED** [review]
nice!! 👍

**Pull request merged**